tests/ses/install_ses.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install SES and deepsea-qa from repo as stable or git as latest unstable code
+# Maintainer: Jozef Pupava <jpupava@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use utils 'zypper_call';
+use version_utils 'is_sle';
+
+sub run {
+    select_console 'root-console';
+    my $git_deepsea = get_var('GIT_DEEPSEA');
```

Comment:
get_var can give you "default" value `get_var('GIT_DEEPSEA', 'SUSE/DeepSea.git');`

Reply:
true, forgot about that, thank you

BTW there is a lot more changes coming here, don't worry with review while in WIP ^^